<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>445</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>433</width>
    <height>342</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>435</width>
    <height>9999</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(240, 240, 240);</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 36, 53);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>THIS PROGRAM IS FOR INTERNAL USE OF
THE COMPANY SISTEMI INDUSTRIALI S.p.a</string>
    </property>
   </widget>
   <widget class="QLabel" name="warning">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="concerning">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>90</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 36, 53);
color: rgb(255, 255, 255);
font: 75 15pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>421</width>
     <height>291</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Basic Search</string>
    </attribute>
    <widget class="QRadioButton" name="radio_seismic">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>64</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>SEISMIC</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_snow">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>SNOW</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_transport">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>130</y>
       <width>83</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>TRANSPORT</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_blasting">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>190</y>
       <width>154</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>BLASTING</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_thermal">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>160</y>
       <width>154</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>THERMAL</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_wind">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>154</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>WIND</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>381</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>PLEASE SELECT A ONE LOADING STATUS OF THOSE PRESENT IN THE 
STRUCTURE TO PERFORM THE SEARCH:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="basic">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>230</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Process</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>421</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TURBINE MODEL:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="turbine_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>381</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>421</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Advanced Search</string>
    </attribute>
    <widget class="QComboBox" name="expo">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <item>
      <property name="text">
       <string>OUTDOOR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>INDOOR</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>140</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>STRUCTURAL CONFIGURATION</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>110</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TURBINE MODEL</string>
     </property>
    </widget>
    <widget class="QComboBox" name="configuration">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <item>
      <property name="text">
       <string>STRUTTURA CON CONTROVENTI</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>STRUTTURA SENZA CONTROVENTI</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="turbine">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>80</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>INSTALLATION TYPE</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>SELECT THE CHARACTERS OF THE ENCLOSURE AS REQUEST BELOW</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>170</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>EXPOSITION TYPE</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>50</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>TYPE OF ENCLOSURE</string>
     </property>
    </widget>
    <widget class="QComboBox" name="enclosure">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <item>
      <property name="text">
       <string>ON-BASE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OFF-BASE</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="process">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>230</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Process</string>
     </property>
    </widget>
    <widget class="QComboBox" name="install">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <item>
      <property name="text">
       <string>ON-SHORE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OFF-SHORE</string>
      </property>
     </item>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>209</y>
       <width>421</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>STRUCTURES SEARCH ENGINE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="all">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>410</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>See All</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
